packages/teams-js/devdocs/architecture.md

```diff
@@ -40,7 +40,7 @@ This option should only be used for work that meets ALL of the below requirement
 
 Here are the steps for adding an API that utilizes version checks (e.g. `if (!isCurrentSDKVersionAtLeast(captureImageMobileSupportVersion)...`):
 
-1. Add the API as a new capability or subcapability rather than adding to an existing capability. Please look at other capabilities such as [calendar.ts](packages/teams-js/src/public/calendar.ts) for examples of how to structure a capability. There must be an isSupported() function with every capability which is a simple boolean check for seeing if `runtime.supports` contains the capability.
+1. Add the API as a new capability or subcapability rather than adding to an existing capability. Please look at other capabilities such as `calendar.ts` for examples of how to structure a capability. There must be an isSupported() function with every capability which is a simple boolean check for seeing if `runtime.supports` contains the capability.
 
 e.g.
 
```

Comment:
Next line in TypeScript not json.

Reply:
There are several code blocks in here that are mistyped as json and should be TypeScript (GitHub is not letting me highlight the line numbers 👎 ). Lines 57 and 76.

Reply:
They could be javascript vs TypeScript, I'm not sure if it matters.